### PR TITLE
touchcursor: Add version 1.7.1

### PR DESCRIPTION
**Pull request by g-berthiaume**
[Touchcursor](https://martin-stone.github.io/touchcursor) is a hidden gem for windows programming productivity.

> **TouchCursor** lets you use the home keys as cursor keys – in all Windows programs – keeping your fingers in the best position for fast typing. It's free and open-source.

Let me know if you see something to change.

### CODE REVIEW

**Comment by rashil2000** [comment]
Last release in 2010??? This is very out of date.

**Comment by g-berthiaume** [comment]
> Last release in 2010??? This is very out of date.

I'm not sure what you mean by that.

This is a compiled C++ program using as only dependency the Win32 api; As long as windows exist, this software will work.
To me, this is what the epitome of good software looks like.

**Comment by rashil2000** [comment]
I'm not saying it's bad.

But software that isn't patched regularly can be vulnerable. Putting it in an official bucket exposes users to risks.

**Comment by g-berthiaume** [comment]
Ah ok. I see what you mean.

Looking at the repo, the author seems to be still active.
https://github.com/martin-stone/touchcursor/issues

That said, it's true it has had update in years.

**Comment by g-berthiaume** [comment]
Your call. Your the maintainer.
I just though people would like, it so I created a PR. :)

**Comment by rashil2000** [comment]
Let me ask some more maintainers about it. @issaclin32 @pratikpc thoughts?

File: bucket/touchcursor.json
Comment (pratikpc) [inline bucket/touchcursor.json:17]: Why not checkver: github?

**Review by pratikpc — CHANGES_REQUESTED** [review]

File: bucket/touchcursor.json
Comment (g-berthiaume) [reply bucket/touchcursor.json:17]: I can certainly do that if you prefer the manifest to parse github releases.

Comment (issaclin32) [reply bucket/touchcursor.json:17]: Actually we can just remove `checkver` and `autoupdate`. The software has not been updated for a few years, so we don't need to check it. This saves resources (i.e. number of connections) for out *Excavator* bot.

**Comment by issaclin32** [comment]
@rashil2000 I terms of security, I am fine with this package.
There are [source code](https://github.com/martin-stone/touchcursor/tree/master/touchcursor) available. We can see it only leverages Windows API to do its tasks.

If it is a software that involves inbound connections, e.g. `apache`, `filezilla-server`, `mercury`, `megacmd`. I will definitely say no. But for this one, I think it is okay.

**Comment by rashil2000** [comment]
Okay, all good from me!

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**
